packages/headings/LinkedHeading.jsx

```diff
@@ -37,15 +50,24 @@ export function LinkedHeading({
           data-test="cta"
           href={ctaLink}
           target={ctaNewWindow ? "_blank" : undefined}
-          className="su-flex su-no-underline hover:su-underline hover:su-text-digital-red dark:hover:su-text-dark-mode-red su-transition su-items-center md:su-items-end md:su-mb-8 lg:su-mb-12 su-text-black dark:su-text-white su-flex-nowrap su-gap-20 md:su-gap-13 su-align-baseline su-text-19"
+          className={cnb(
```

Comment:
I made some alignment adjustment to the LinkedHeading - The baseline alignment seems to look best. I want to ask you all if you prefer this before I increment the version on the other components that use this LinkedHeading package so currently, the LinkedHeadings on the homepage will look misaligned on the DEV site.